kubernetes/operationcode_backend/base/deployment.yaml

```diff
@@ -112,7 +112,7 @@ spec:
               name: backend-secrets   
       - name: sidekiq
         image: operationcode/operationcode_backend:latest
-        command: ["sidekiq", "-C config/sidekiq.yml.erb"]
+        command: ["bundle", "exec", sidekiq", "-C config/sidekiq.yml.erb"]
```

Comment:
You're missing an opening quote on `sidekiq`.

Reply:
Fixed.